browser-test/src/admin/admin_predicates.test.ts

```diff
@@ -693,40 +693,40 @@ test.describe('create and edit predicates', () => {
     test.beforeEach(async ({page, adminQuestions}) => {
       await loginAsAdmin(page)
 
-      // DATE, STRING, LONG, LIST_OF_STRINGS, LIST_OF_LONGS
-      await adminQuestions.addNameQuestion({questionName: 'single-string'})
-      await adminQuestions.addTextQuestion({questionName: 'list of strings'})
-      await adminQuestions.addNumberQuestion({questionName: 'single-long'})
-      await adminQuestions.addNumberQuestion({questionName: 'list of longs'})
+      await adminQuestions.addNameQuestion({questionName: 'name-question'})
+      await adminQuestions.addTextQuestion({questionName: 'text-question'})
+      await adminQuestions.addNumberQuestion({
+        questionName: 'number-question-equal-to',
+      })
+      await adminQuestions.addNumberQuestion({
+        questionName: 'number-question-one-of',
+      })
       await adminQuestions.addCurrencyQuestion({
-        questionName: 'predicate-currency',
+        questionName: 'currency-question',
       })
       await adminQuestions.addDateQuestion({
-        questionName: 'predicate-date-is-earlier-than',
+        questionName: 'date-question-is-earlier-than',
       })
       await adminQuestions.addDateQuestion({
-        questionName: 'predicate-date-on-or-after',
+        questionName: 'date-question-on-or-after',
       })
       await adminQuestions.addDateQuestion({
-        questionName: 'predicate-date-age-older-than',
+        questionName: 'date-question-age-older-than',
       })
       await adminQuestions.addDateQuestion({
-        questionName: 'predicate-date-age-younger-than',
+        questionName: 'date-question-age-younger-than',
       })
       await adminQuestions.addDateQuestion({
-        questionName: 'predicate-date-age-between',
+        questionName: 'date-question-age-between',
       })
       await adminQuestions.addCheckboxQuestion({
-        questionName: 'both sides are lists',
+        questionName: 'checkbox-question',
```

Comment:
question (non-blocking):  Are we losing some information in the name about both sides being lists (whatever that means)?

Reply:
This string still appears where the test actually configures the predicate, which is what it's referring to, so I think it's actually clearer to remove it here because it is confusing. :)

Thanks!